src/storybook/stand-alone-documentaion/typography/typography-migration-guide.stories.mdx

```diff
@@ -0,0 +1,106 @@
+import { Meta } from "@storybook/addon-docs";
+import { Link } from "vibe-storybook-components";
+import { TEXT, HEADING, EDITABLE_HEADING } from "../../components/related-components/component-description-map";
+import { migrationTable } from "./assets";
+
+<Meta title="Text/Typography Migration Guide" />
```

Comment:
I think it should be on the root level of the documentation, below the changelog